pkg/controller/servicebinding/secret.go

```diff
@@ -27,10 +32,68 @@ func (s *secret) buildResourceClient() dynamic.ResourceInterface {
 	return s.client.Resource(gvr).Namespace(s.ns)
 }
 
-// createOrUpdate will take informed payload and either create a new secret or update an existing
-// one. It can return error when Kubernetes client does.
-func (s *secret) createOrUpdate(payload map[string][]byte, ownerReference metav1.OwnerReference) (*unstructured.Unstructured, error) {
+// compare existing secret and new secret
+func (s *secret) isSame(secretName string, payload map[string][]byte) bool {
```

Comment:
Are you planning to include unit tests for `isSame`?

Reply:
No. The method is not used anywhere. 
Removed it here a3409c3